src/test/fixtures/apps/cms/api/v2/pages/%3Ftype%3Dbooks.Book%26fields%3Dcnx_id%2Cauthors%2Cpublish_date%2Ccover_color%2Camazon_link%2Cbook_state%26slug%3Dbook-slug-1

```diff
@@ -0,0 +1,19 @@
+{
+    "meta": {
+        "total_count": 1
```

Comment:
Just curious, how was this file created? How did the other one in fixtures got modified?

Reply:
This is just a normal file that you can create with `nano name` 
The other file in this folder was a symbolic link to this file `ln -s source/file link/file`
I've decided to use a normal file instead of a symbolic link because editing symbolic links and pushing them is kind of broken and more time consuming than just copying the content of the file that is next to this one